### DISCUSSION TITLE
I cannot install Cucumber report to my webdriverio dependency

### DISCUSSION THREAD

**Discussion by QuocBaoNguyen11295** [webdriverio/webdriverio#9824] [Community]
Hi Community
I've tried to run `npm install wdio-cucumberjs-json-reporter --save-dev` dependency to my project
But after running this dependency. I got the following message:
`$ npm install wdio-cucumberjs-json-reporter --save-dev
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: webdriverio_cucumber@1.0.0
npm ERR! Found: @wdio/cucumber-framework@8.3.0
npm ERR! node_modules/@wdio/cucumber-framework
npm ERR!   dev @wdio/cucumber-framework@"^8.3.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @wdio/cucumber-framework@"7.x" from wdio-cucumberjs-json-reporter@4.4.3
npm ERR! node_modules/wdio-cucumberjs-json-reporter
npm ERR!   dev wdio-cucumberjs-json-reporter@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.`

Have you ever found this error yet?
If you have found this. Could you give me a solution to solve this?
Many thanks to you (^_^)

**Comment by christian-bromann** [comment]
see https://github.com/webdriverio-community/wdio-cucumberjs-json-reporter/issues/102

**Reply by QuocBaoNguyen11295** [reply]
Thank you @christian-bromann 
I will take a look on this
